### DISCUSSION TITLE
Backend footer change

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2428] [Ideas]
This is the footer in OpenMage Backend

![image](https://user-images.githubusercontent.com/8360474/184619829-1162eb9e-985f-445e-9780-6c59e0539aba.png)

It could look better. On the left, I would leave only the language selector in the interface. I would move both blue links to the right and change their names.

OpenMage Community | Report a Bug

![new_look](https://user-images.githubusercontent.com/8360474/184620621-eb751bdd-b0f7-4bdd-a5ed-2298acefe273.jpg)

**Comment by fballiano** [comment]
Your version it's so much better!
Can I make a PR out of it?

**Reply by addison74** [reply]
If I started doing PR, I would forget the ideas that come to me at the moment. That's why I throw them through all kinds of posts.

If you liked the idea please feel free to do the implementation.

**Reply by fballiano** [reply]
I think it's much more clean, I'll try to code it asap :-)

**Reply by addison74** [reply]
Here we have a functionality issue. As a rule, links are not placed over the selections because there are two actions there in a small area, it distracts the attention, there is no clarity. People today are no longer so attentive to details and for this reason we need to air the interface as much as possible. We realize the true advantages when the old version is put back in place.

**Comment by addison74** [comment]
This was implemented in OpenMage.
